core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6562WarnDefaultBindings.java

```diff
@@ -43,7 +43,8 @@ public void testItShouldNotWarn()
         verifier.setAutoclean( false );
         verifier.setLogFileName( phase + ".txt" );
         verifier.setForkJvm( true ); // required due to --fos-on-severity
```

Comment:
Can you change that to `fail-on-security`? It seems wrong here.

Reply:
I've taken the liberty to go for `fail-on-severity` ;-)